compaction/compaction_manager.hh

```diff
@@ -284,6 +289,10 @@ public:
         return _stats;
     }
 
+    // gets the table's compaction state
+    // throws std::out_of_range exception if not found.
+    compaction_state& get_compaction_state(table* t);
```

Comment:
We need a follow-up cleanup to use `table` consistently in `compaction_manager`. Half the methods here use `column_family` and the other half uses `table`. It makes it look like there are two different classes involved here.

Reply:
Agreed.  It contains historical code.
Cc @raphaelsc